cmd/kube-apiserver/app/server.go

```diff
@@ -51,7 +51,8 @@ import (
 const (
 	// Maximum duration before timing out read/write requests
 	// Set to a value larger than the timeouts in each watch server.
-	ReadWriteTimeout = time.Minute * 60
+	ReadWriteTimeout            = time.Minute * 60
+	defaultLongRunningRequestRE = "(/watch$|/proxy(/.*|))"
```

Comment:
* can't assume `/watch` terminates the string
* `(/.*|)` without an anchor in the second half is meaningless